php/README.md

```diff
@@ -14,10 +14,14 @@ generation functionality.
 
 ## Requirements
 
-To use PHP runtime library requires:
+Using the PHP runtime library requires:
 
-- C extension: PHP 7.x, 8.0
-- [PHP package](http://php.net/downloads.php): PHP 5.5, 5.6, 7.x, or 8.0.
+-   C extension: PHP 7.x, 8.x
```

Comment:
In the news article that should be published today: "As per our official [PHP support policy](https://cloud.google.com/php/getting-started/supported-php-versions), we will be dropping support for PHP 7.4 and lower. This means the minimum supported PHP version is 8.0."